src/git/config.go

```diff
@@ -59,15 +34,33 @@ func EnsureIsFeatureBranch(branchName, errorSuffix string) {
 }
 
 // GetAncestorBranches returns the names of all parent branches for the given branch,
-// beginning but not including the parennial branch from which this hierarchy was cut.
```

Comment:
This was wrong as it does include the perennial branch from which this hierarchy was cut

Reply:
good catch